package.json

```diff
@@ -9,7 +9,8 @@
     "node": "^10.x"
   },
   "scripts": {
-    "build": "yarn && yarn test && NODE_ENV=production webpack -p --bail --progress",
+    "build": "yarn && yarn test && yarn prod",
+    "prod": "NODE_ENV=production webpack -p --bail --progress",
```

Comment:
Unrelated but ok, let's keep this

Reply:
I'm not sure what the point of this change is? Tests shouldn't be skipped when building out when it's ready to commit, and if developers want to skip tests during development that's what the `dev` script is for.

Reply:
@wilr Is this needed? I'd prefer to keep this as it was (too likely someone will use `prod` instead of `build` when pushing their PR).

Reply:
I retract my previous comment. Let's revert this.

Reply:
@GuySartorelli reverted that change now. I did that as the test fails for me. Because the tests fail the build doesn't execute. Some error locally about AnchorField.